docs/configuration.md

```diff
@@ -378,6 +378,18 @@ various parts of the Emacs integration.
 In particular, `lsp-haskell` provides customization options for the `haskell-language-server`-specific parts,
 such as the path to the server executable.
 
+#### Vanilla Emacs
+
+If you're using `lsp-mode` generic plugin configuration can be added using `lsp--set-configuration`:
+
+```emacs-lisp
+(add-hook 'lsp-after-initialize-hook
+          '(lambda ()
+             (lsp--set-configuration
+              `(:haskell (:plugin (:tactics (:config (:timeout_duration 5)))))
```

Comment:
Out of curiosity, why do you use backquote/quasiquote here instead of normal quote?

Reply:
TBH it took me a while to find the right concoction, and, this was just some cruft that was left, it may not need it, but it's where my configuration stood when it started working!

Reply:
I think `'(:haskell (:plugin (:tactics (:config (:timeout_duration 5)))))` should also work. Could you try that, and if it work, use it instead? I think, for document, it would be better to be consistent (your previous lines use `'` quote).

Reply:
done!